src/backend/distributed/deparser/ruleutils_13.c

```diff
@@ -2091,13 +2094,24 @@ get_select_query_def(Query *query, deparse_context *context,
 	}
 	if (query->limitCount != NULL)
 	{
-		appendContextKeyword(context, " LIMIT ",
-							 -PRETTYINDENT_STD, PRETTYINDENT_STD, 0);
-		if (IsA(query->limitCount, Const) &&
-			((Const *) query->limitCount)->constisnull)
-			appendStringInfoString(buf, "ALL");
-		else
+		if (query->limitOption == LIMIT_OPTION_WITH_TIES)
+		{
+			// had to add '(' and ')' here because it fails with casting
```

Comment:
I wonder how come `citus_indent` is ok with `//` comments here?

Reply:
Because we ignore this file for citus_indent: https://github.com/citusdata/citus/blob/b271a4890b5638fae5135ce8ef12df16adb07d02/.gitattributes#L32